1.0/files/config.ini

```diff
@@ -0,0 +1,755 @@
+; version=8
```

Comment:
Did you update this file? Probably not, so could you please do that? @Crabbey

Reply:
Did something changed in there? I didn't noticed anything.

Reply:
I am not sure if something did. Thats why we need to check it.